versioned_docs/version-5.x/faq/migrating-from-ink-4-to-5.md

```diff
@@ -585,3 +585,46 @@ You can find a contract example and a comparison with chain extensions
 [here](https://github.com/paritytech/ink/tree/master/integration-tests/call-runtime).
 We've added an example of how to end-to-end test
 `call_runtime` [here](https://github.com/paritytech/ink/tree/master/integration-tests/e2e-call-runtime).
+
+
+### `call_v2`
+
+There is a new host function `call_v2` which allows passing both `Weight` parts: 
+`ref_time_limit` and `proof_time_limit` and the `storage_deposit_limit`. 
+The legacy call function only provides the single `gas_limit` parameter, 
+which is used as the value for `ref_time_limit`.
+
+These parameters can be set on a call builder instance, by first calling `call_v2()` e.g.
```

Comment:
As @smiasojed says, this line is incorrect (I can see it comes from the description in https://github.com/paritytech/ink/pull/2077 which I have now corrected). 

Calling `call_v2` on the call builder is no longer required, it calls into the underlying `ext::call_v2` host function now by default. Now, in order to use the legacy `ext::call` method you need to call `call_v1` on the builder. I see you have already mentioned this below.